### DISCUSSION TITLE
Submitting project proposals for GSoC 2023

### DISCUSSION THREAD

**Discussion by kmehant** [cncf/mentoring#775] [Google Summer of Code 2023]
Hi Team:

When could we submit project proposals for GSoC 2023? 

I see that this link is for LFX mentorship: https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2023/01-Mar-May

Looking forward to hear.

**Comment by nate-double-u** [comment]
Sorry I missed this earlier. I see you’ve already submitted a PR, but for completeness (and for those looking for the answer in future): we’ll make announcements here in the Discussions boards when we open applications. This year’s call for proposals is here: https://github.com/cncf/mentoring/discussions/777
